expat/lib/xmlparse.c

```diff
@@ -736,6 +736,13 @@ writeRandomBytes_arc4random(void *target, size_t count) {
 
 #ifdef _WIN32
 
+/* Provide declaration of rand_s() for MinGW-32 (not 64, which has it),
+   as it doesn't declare it in its header up to at least 5.2.2 version
+   of its runtime. */
```

Comment:
`rand_s()` is (supposed to be) declared in `stdlib.h`, which is is not part of WSL headers, but rather of `mingwrt-xxx` package, so I don't think this would be correct. I've tried to reword it more carefully, but please feel free to update it as you see fit.

Reply:
It's not w32api — you're right!